src/os/bluestore/bluefs_types.h

```diff
@@ -9,9 +9,9 @@
 
 class bluefs_extent_t : public AllocExtent{
 public:
-  uint16_t bdev;
+  uint8_t bdev;
```

Comment:
Suggest to update BlueFS::_allocate method signature as well to pass uint8_t for id parameter.